index.js

```diff
@@ -54,6 +57,18 @@ const encoder = {
     return Buffer.concat([tf, d])
   },
 
+  blob(blobId) {
+    let tf
+    if (blobId.endsWith('sha256')) tf = CLASSIC_BLOB_TF
```

Comment:
Pedantic comment but could we rather match on `endsWith('.sha256')` (dot included) to keep consistent with the other suffix matches?

Reply:
Well spotted!